src/pipecat/services/ai_services.py

```diff
@@ -115,28 +115,38 @@ def __init__(self, **kwargs):
         self._start_callbacks = {}
 
     # TODO-CB: callback function type
-    def register_function(self, function_name: str, callback, start_callback=None):
+    def register_function(self, function_name: str | None, callback, start_callback=None):
+        if not function_name: # then use this callback for all functions
+            function_name = "*" 
```

Comment:
No need to use `*`, in Python you can use `None` keys. So, no need to do anything special.